libs/SalesforceKit/SalesforceKit.modulemap

```diff
@@ -4,16 +4,6 @@ framework module SalesforceKit {
     export *
     module * { export * }
     
-    framework module SalesforceCommonUtils {
-        umbrella header "SalesforceCommonUtils.h"
-        
-        export *
-        module * { export * }
-        
-        link "System"
-        link "z"
-    }
-    
     framework module SalesforceOAuth {
```

Comment:
While updating this file I noticed that ```SalesforceKit``` doesn't include ```SalesforceReact``` yet. We can talk offline about updating/maintaining this file, especially since ```SalesforceKit``` is hidden/not used in our workspace.